src/test/java/tech/jhipster/lite/generator/server/springboot/dbmigration/liquibase/application/LiquibaseApplicationServiceIT.java

```diff
@@ -83,6 +85,39 @@ void shouldAddChangelogMasterXml() {
     assertFilesLiquibaseChangelogMasterXml(project);
   }
 
+  @Test
+  void shouldAddChangelog() {
+    Project project = tmpProject();
+
+    liquibaseApplicationService.addChangelogMasterXml(project);
+    liquibaseApplicationService.addChangelogXml(project, "v1", "master.xml");
+
+    String expected = "  <include file=\"config/liquibase/changelog/v1/master.xml\" relativeToChangelogFile=\"false\"/>";
+
+    assertFileContent(project, getPath(MAIN_RESOURCES, "config/liquibase/master.xml"), expected);
```

Comment:
To be sure that the NEEDLE is still here :)

```suggestion
    assertFileContent(project, getPath(MAIN_RESOURCES, "config/liquibase/master.xml"), List.of(expected, NEEDLE_LIQUIBASE));
```